src/Compilers/Core/Portable/InternalUtilities/FatalError.cs

```diff
@@ -20,8 +20,12 @@ namespace Microsoft.CodeAnalysis.ErrorReporting
 {
     internal static class FatalError
     {
+        // A delegate type for our NonFatalHandler; this is necessary since this file must build on .NET 2.0 where
```

Comment:
This file must build on .NET 2.0...? Who? How? Why?

Reply:
The EE result provider

Reply:
We build some debugger components in Roslyn against 2.0. I'm sure @jaredpar can tell you more. :smile:

Reply:
Do you like scary stories?

Reply:
I had to make another change: we're now using `Action<T1, T2>` and this code is simply #if'ed out to not build in the EE ResultProvider.